uMatriks/ChatRoom.qml

```diff
@@ -25,12 +25,23 @@ Rectangle {
 
     function sendLine(text) {
         console.log("Sending: " + text)
-        console.log("Room: " + currentRoom)
-        console.log("Conn: " + currentConnection)
+        // console.log("Room: " + currentRoom)
+        // console.log("Conn: " + currentConnection)
+        if (text.trim().length === 0) {
+            return
+        }
         if(!currentRoom || !currentConnection) {
             return
         }
-        currentConnection.postMessage(currentRoom, "m.text", text)
+
+        var type = "m.text"
+        var PREFIX_ME = '/me '
+        if (text.indexOf(PREFIX_ME) === 0) {
+            text = text.substr(PREFIX_ME.length)
+            type = "m.emote"
+        }
+
+        currentConnection.postMessage(currentRoom, type, text)
```

Comment:
Better use `currentRoom.postMessage(type, text)` here, `Connection::postMessage` is deprecated these days.

Reply:
Done